README.md

```diff
@@ -1,2 +1,13 @@
-# banidb-api-python
-Python bindings for bandidb API
+# BaniDB.[py](https://pypi.org/user/KhalisFoundation/)
+### _Python package for BaniDB API_
+
+[<img height=100 src='bdb.svg'>](http://banidb.com)

```

Comment:
This does not display on my mac markdown reader.

This one does - 
`![](bdb.svg) `